packages/app-bridge/src/AppBridgeTheme.ts

```diff
@@ -77,6 +83,7 @@ export type ThemeContext = {
     isPublicLink: boolean;
     isAuthenticated: boolean;
     languages: Language[];
+    template: TemplateContext | null;
```

Comment:
Why would it be `null`?

Reply:
it can be instantiated without this data, then populate it with the template data.

Reply:
Why can't you instantiate it with data?

Reply:
Because it is `AppBridgeTheme` is instantiated in other components outside themes (`powerbar.ts` for example) or where `context.template` info is not needed. It doesn't make sense to make at least 1 api request to fulfill the template context if it's not needed.